src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
```

Comment:
your solution is good. But if i look at this code in few weeks, i think i cant understand all these numbers and why we substraction them. maybe you can do more variables?